distribution/std-lib/Standard/src/Visualization/Scatter_Plot.enso

```diff
@@ -0,0 +1,146 @@
+from Standard.Base import all
```

Comment:
You're missing type signatures for most of this file. Also, many things here could be done as extension methods rather than module-level statics. Again, the former is more idiomatic.